test/e2e/common/deploy.go

```diff
@@ -110,13 +113,22 @@ func DeployCSI(f *framework.Framework, additionalInstallArgs string) (func(), er
 		podWait         = 3 * time.Minute
 		sleepBeforeWait = 10 * time.Second
 		schedulerRC     = newSchedulerRestartChecker(f.ClientSet)
+		isRestarted     = false
+		err             error
 	)
 
 	if additionalInstallArgs != "" {
 		installArgs += " " + additionalInstallArgs
 	}
 
 	cleanup := func() {
+		// Collect and print Operator logs
+		if logs, err := GetOperatorLogs(f.ClientSet); err != nil {
```

Comment:
Can we see operator logs in runtime?

Reply:
reworked, CSI Operator deploys in the same namespace